coops-ui/src/views/StandaloneDirectorsFiling.vue

```diff
@@ -348,7 +356,8 @@ export default {
       totalFee: 0,
       // EntityTypes Enum
       EntityTypes,
-      FilingCodes
+      FilingCodes,
+      complianceDialogMsg: null
```

Comment:
This looks like complianceDialogMsg should be an "EntityTypes Enum" (per comment on line 357).

Please move this up with the other properties or add a blank line above it to start a new properties section.